.github/workflows/terraform-observe_scheduler.yaml

```diff
@@ -15,38 +15,31 @@ jobs:
     with:
       skip: ${{ inputs.skip }}
 
-  get-test-directories:
-    if: "!contains(fromJSON(inputs.skip).jobs, 'terraform-tests')"
-    name: Get Terraform Test Directories
+  find-modules:
     runs-on: ubuntu-latest
     steps:
-      - name: Checkout
-        uses: actions/checkout@v3
-      - name: Build matrix
-        id: matrix
-        shell: bash
-        run: |
-          DIRS="$(find . -path "*tftests/*" ! -path "*.terraform*" -type d )"
```

Comment:
The peril of these sketchy shell scripts...

This is meant to match any child directory `tftests`...

Reply:
https://github.com/bendrucker/find-terraform-modules/pull/173